crates/cli-tools/src/changelog.rs

```diff
@@ -228,7 +243,11 @@ pub async fn execute_ci() -> Result<()> {
         let path = path?;
 
         // Validation done during parsing.
-        Changelog::read_file(&path).await?;
+        let changelog = Changelog::read_file(&path).await?;
+
+        let crate_dir = path.strip_suffix("/CHANGELOG.md").context("Malformed file path")?;
```

Comment:
Same, I think we can `unwrap()` here. This is not a user error, this is an internal bug.